Add-ons/UmbracoForms/Developer/Extending/Adding-a-Type.md

```diff
@@ -199,3 +199,22 @@ public class TextareaWithCount : Umbraco.Forms.Core.Providers.FieldTypes.Textare
     }
 }
 ```
+
+In addition to the above, you must also register the extended field type within a composer and also create the backoffice field type view.
```

Comment:
```suggestion
As discussed in the previous section, you must also register the extended field type within a composer.  You also need to create the the backoffice field type view.
```

This is partly covered in the section above, so is slightly repeated here.  No harm in that though, so suggest we keep but relate it to the previous copy.